docs/API/LorisRESTAPI.md

```diff
@@ -696,3 +696,76 @@ The JSON object is of the form:
     "Value" : string
 }
 ```
+
+# 5.0 DICOM Data
+
+Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$Visit` 
+portion of the REST API namespaces, but is defined in a separate section of this 
```

Comment:
namespace is singular.

Reply:
fixed; I also fixed it in an earlier line as well (line 440)